addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/discovery/AVMFritzUpnpDiscoveryParticipant.java

```diff
@@ -23,22 +23,24 @@
 import org.jupnp.model.meta.ModelDetails;
 import org.jupnp.model.meta.RemoteDevice;
 import org.openhab.binding.avmfritz.BindingConstants;
+import org.osgi.service.component.annotations.Component;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * The {@link AvmUpnpDiscoveryParticipant} is responsible for discovering
+ * The {@link AVMFritzUpnpDiscoveryParticipant} is responsible for discovering
  * new and removed FRITZ!Box devices. It uses the central
  * {@link UpnpDiscoveryService}.
  *
  * @author Robert Bausdorf - Initial contribution
  * 
  */
-public class AvmUpnpDiscoveryParticipant implements UpnpDiscoveryParticipant {
+@Component(service = UpnpDiscoveryParticipant.class, immediate = true, configurationPid = "discovery.avmfritz")
```

Comment:
This is a UPnP discovery participant - it does not make sense to pass configurations with pid "discovery.avmfritz" to it as it itself does not implement any background discovery (it is the UPnP service, which does it).